doc/examples/transform/plot_register_align.py

```diff
@@ -0,0 +1,73 @@
+from skimage.data import camera
```

Comment:
Can you organise and group your import statements here according to the PEP8 recommndations? Thanks!

That is:
1. Standard library imports
2. Third party library imports
3. Imports from scikit-image
(with a blank line between each group).
https://www.python.org/dev/peps/pep-0008/#imports